.env.example

```diff
@@ -123,3 +123,10 @@ DPID_URL_OVERRIDE=https://dev-beta.dpid.org
 MUTE_PUBLISH_WORKER=false
 # SingleNodeLockServce
 MAX_LOCK_TIME=3600 # 1 hour
+
+DOI_PREFIX=https://doi.org/10.62891
+
+# Cross ref api 
+CROSSREF_API=https://api.crossref.org
+CROSSREF_API_KEY=
+CROSSREF_EMAIL=
```

Comment:
Should it default to `noreply@desci.com`? I think you mentioned the API erroring if this wasn't passed.

Reply:
No, Only the plus token errors when we use a placeholder value